api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -146,6 +147,7 @@ public void setLogging(Logging logging) {
         this.logging = logging;
     }
 
+    @PresentInVersions("v1alpha1-v1beta1")
```

Comment:
What's the syntax for the value of this annotation? It's an interval or a list? or something different?

Reply:
It's described here: https://github.com/strimzi/strimzi-kafka-operator/pull/3653/files#diff-762d04edf28bcda2d851e559e33f5ea4R93. In this case it's an interval with inclusive ends (we don't actually know whether there is a v1alpha2, we just compute inclusion based on the endpoints). It can also be an open-ended interval (e.g. `v1beta2+`), or a single version (`v1beta1`).

Reply:
Thanks, got it!